src/sidebar/panel.js

```diff
@@ -118,7 +118,7 @@ ClassicEditor.create(document.querySelector('#editor'), {
             waitingToReconnect = false;
             clearTimeout(loginTimeout);
             // set title attr of footer to the currently logged in account
-            footerButtons.title = eventData.profile && eventData.profile.email;
+            footerButtons.title = `${browser.i18n.getMessage('syncToMail', eventData.profile.email)}`
```

Comment:
Maybe we need to do:

```
if (eventData.profile) {
    footerButtons.title = `${browser.i18n.getMessage('syncToMail', eventData.profile.email)}`
}
```

However since #637 landed you might want grab the address differently